addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -1445,13 +1579,13 @@ export class Rtc {
             return;
         }
         const transceivers = rtcSession.peerConnection.getTransceivers();
-        const transceiver = transceivers[ORDERED_TRANSCEIVER_NAMES.indexOf("video")];
+        const transceiver = transceivers[ORDERED_TRANSCEIVER_NAMES.indexOf("camera")];
```

Comment:
as @alexkuhn mentioned on discord, this part is incomplete:

This function's goal is to turn transceivers on/off based on whether their output is actually consumed, for example if there is 0 cards that show a video, the transceiver for that video is turned off to save bandwidth.

Currently, `rtcSession.videoCompomentCount` represents this (how many `callParticipantCard`s are in the DOM), and it is incremented (/decremented) on `callParticipantCard`'s `mount`/`willUnmount`.

But now that there are two different kind of video card (and transceivers) there should be two fields on `rtcSession`, one `cameraComponentCount` and one `screenCompomentCount`, and this function should take a third argument `videoType`<"video"|"camera"> to know which one to increment.

And then `getTransceiverDirection()` should be changed too so it can return the appropriate direction based on the type and count.

If you do the change in my comment https://github.com/odoo/odoo/pull/130354#discussion_r1327053192, you will have a `videoType` in `CallParticipantCard.props.cardData` and can pass it when`updateVideoDownload()` is called.

Reply:
Relatively important to fix this problem before merge.
I would do idiomatic changes to fix the issue without code cleaning suggested in https://github.com/odoo/odoo/pull/130354#discussion_r1327053192, if this requires a lot of diff.

Reply:
@alexkuhn yes you're right, it can be done without https://github.com/odoo/odoo/pull/130354#discussion_r1327053192, the proper implementation of `getTransceiverDirection()` will likely need the cards to be aware of their video type which can also be useful for the other comment change.

Reply:
I've only made a small change related to the proper transceivers directions.